File: first_third/linear.js
function basicLinear(str, str2) {
    let arr = str2.split('');
    for (let i=0; i<arr.length; i++) {
        if (str === arr[i]) {
            return i
        }
    }
}

console.log(basicLinear('a', 'banana'))

function globalLinear(str, str2) {
    let data = []
    let arr = str2.split('');
    for (let i=0; i<arr.length; i++) {
        if (str === arr[i]) {
            data.push(i)
        }
    }
    return data
}

console.log(globalLinear('a', 'banana'))
console.log(globalLinear('n', 'banana'))
console.log(globalLinear('5', '15845321565184'))
console.log(globalLinear(' ', "How you doin'?"))

File: first_third/linear.py
def basic_linear(str, str2):
    lst2 = []
    for i in range(len(str2)):
        if str == str2[i]:
            return i


print(basic_linear("a", "banana"))

def globalLinear(str, str2):
    data = []
    for i in range(len(str2)):
        if str == str2[i]:
            data.append(i)
    return data
print(basic_linear("a", "banana"))
print(globalLinear('n', 'banana'))
print(globalLinear('5', '15845321565184'))
print(globalLinear(' ', "How you doin'?"))